README.md

```diff
@@ -123,6 +124,36 @@ To enable this feature, simply update your options to include `PDO::ATTR_PERSIST
 ]) : [],
 ```
 
+## `ORDER BY` Clause in `DELETE` and `UPDATE` Queries
+
+SingleStore does not support the `ORDER BY` clause in the `DELETE` and `UPDATE` queries.
+Issuing queries similar to the following will return an error. 
+
+```php
+DB::table('test')->orderBy('id', 'asc')->update(['id' => 1, 'a' => 'b']);
+DB::table('test')->orderBy('id', 'asc')->delete();
+```
+
+You can configure the driver to ignore `orderBy` in `delete()` and `update()` requests by enabling `ignore_order_by_in_deletes` and
+`ignore_order_by_in_updates` in the connection configuration, respectively. For example:
+
+```php
+[
+    'default' => env('DB_CONNECTION', 'singlestore'),
+
+    'connections' => [
+        'singlestore' => [
+            'driver' => 'singlestore',
+            'ignore_order_by_in_deletes' => true,
+            'ignore_order_by_in_updates' => true,
+            ...
+        ],
+    ]
+]
+```
+
+Note that when `orderBy` is ignored, it may result in deletion/update of different rows.
```

Comment:
Is it possible to be more specific about when this can happen? Is presence of LIMIT/OFFSET the only factor or not?

Reply:
Yep, it is the only factor